src/network/core_utils.rs

```diff
@@ -34,6 +35,65 @@ impl CoreUtils {
         final_slice.join(":")
     }
 
+    pub fn get_macvlan_mode_from_string(mode: &str) -> u32 {
+        // Replace to constant from library once this gets merged.
+        // TODO: use actual constants after https://github.com/little-dude/netlink/pull/200
```

Comment:
PR is merged, you can change it.

Reply:
@Luap99 A version needs to be cut for `netlink-packet-core` or we would need to include it in our `Cargo.toml`. There is a cascading dependency of versioned packages preventing me to use `master`.

Reply:
ok lets keep it for now

Reply:
I'll try finding a better way to point all dependent package to `master` in a new PR maybe if that works.